### PR TITLE
[sample_encode] Replaced mfxBitstream with mfxBitstreamWrapper

### PR DESCRIPTION
**Pull request by dmitryermilov**
Signed-off-by: Dmitry Ermilov <dmitry.ermilov@intel.com>

### CODE REVIEW

File: samples/sample_common/src/sample_utils.cpp [hunk 1]
```diff
@@ -1644,51 +1644,7 @@ mfxU32 GCD(mfxU32 a, mfxU32 b)
     return b1;
 }
 
-mfxStatus DARtoPAR(mfxU32 darw, mfxU32 darh, mfxU32 w, mfxU32 h, mfxU16 *pparw, mfxU16 *pparh)
```
Comment (mgonchar) [inline samples/sample_common/src/sample_utils.cpp]: Shouldn't it be different PR?

Comment (dmitryermilov) [reply samples/sample_common/src/sample_utils.cpp]: Nooo. Consider it a cleanup:)
